save-frontend/src/main/kotlin/com/saveourtool/save/frontend/externals/prograssbar/ReactCircleBuilder.kt

```diff
@@ -0,0 +1,35 @@
+@file:Suppress("HEADER_MISSING_IN_NON_SINGLE_CLASS_FILE")
+
+package com.saveourtool.save.frontend.externals.prograssbar
+
+import react.ChildrenBuilder
+import react.react
+
+private val rad = Pair("rgb(172, 0, 0)", "#ac0000")
+
+private val green = Pair("rgb(0, 213, 0)", "#00d500")
+
+/**
+ * @param handler
+ * @param progress
+ */
+@Suppress("MAGIC_NUMBER")
+fun ChildrenBuilder.circle(
+    progress: Long,
+    handler: ChildrenBuilder.(ReactCircleProps) -> Unit = {},
+) {
+    val color = if (progress < 51) {
```

Comment:
`progress < 51` is green, but else it is `red`? Need to add description on what you are doing in this PR and why. Why would you add this function and how are you planning to use it?

Reply:
Added description

Reply:
Firstly, `MAGIC_NUMBER`
Secondly, some KDoc update is required as once someone would reuse this builder in order to use it as a usual progress bar/circle

Reply:
Removed this logic. I'll add it myself already twist.

Reply:
@Cheshiriks please follow conventions, while you create commits/PRs:
```
### What's done:
- A
- B
- C, more information about C and what's going on
```